File: Client/Client/Model/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace Client.Model
{
    public class Ticket
    {
        public Ticket(int price, string luggage)
        {
            this.price = price;
            this.luggage = luggage;
        }

        public Ticket()
        {
        }

        [JsonPropertyName("price")]
        public int price { get; set; }
        [JsonPropertyName("luggage")]
        public string luggage { get; set; }

        public override string ToString()
        {
            return "Price: " + price + " Luggage: " + luggage;
        }
    }
}

File: Middleware/src/main/java/Shared/Ticket.java
package Shared;

import java.io.Serializable;

public class Ticket implements Serializable
{
public int price;
public String luggage;

  public Ticket(int price, String luggage)
  {
    this.price = price;
    this.luggage = luggage;
  }

  @Override public String toString()
  {
    return "Ticket{" + "price=" + price + ", luggage='" + luggage + '\'' + '}';
  }
}
